cmd/validator/accounts/exit_test.go

```diff
@@ -134,7 +134,7 @@ func TestExitAccountsCli_OK_AllPublicKeys(t *testing.T) {
 
 	mockNodeClient.EXPECT().
 		GetGenesis(gomock.Any(), gomock.Any()).
-		Times(3).
+		Times(2).
```

Comment:
hmm do we need to get genesis twice? since it doesn't change why do we need to get it multiple times? maybe i didn't check how the call should trigger. shouldn't it only be 1 call?